### DISCUSSION TITLE
pick drawing from lower layer

### DISCUSSION THREAD

**Discussion by PetukhovArt** [zouyaoji/vue-cesium#628] [Q&A]
hello ! 
@zouyaoji can you help , how to pick some renderData of lower layer drawing , when the another drawing is above ? 

only the last layer is picking , i tried drillPick with core cesium js event handlers , and it is not working too

**Comment by zouyaoji** [comment]
Try using <vc-selection-indicator ref="selectionIndicator" @pick-evt="pickEvt"></vc-selection-indicator>

https://zouyaoji.top/vue-cesium/#/en-US/component/controls/vc-selection-indicator
